cluster/router/polaris/router.go

```diff
@@ -53,6 +53,12 @@ var (
 )
 
 func newPolarisRouter() (*polarisRouter, error) {
+	if err := remotingpolaris.Check(); errors.Is(err, remotingpolaris.ErrorNoOpenPolarisAbility) {
```

Comment:
Why does this pull request contain code about the Polaris router?

Reply:
when we ues apache:3.0 branch, it contains other's pr

<img width="933" alt="image" src="https://user-images.githubusercontent.com/23113152/216988192-92b5ad9f-caed-4769-aac9-3a5340ec11fb.png">

Reply:
It sounds like you select the wrong branch when opening this pull request. I suggest you:

1. Close this pull request.
2. Checkout to the 3.0 branch and pull the code from the upstream.
3. Rebase your branch on the latest 3.0 branch.
4. Open a new pull request with the right target branch (3.0 at this time).

Reply:
okay